app/controllers/usersController.js

```diff
@@ -29,4 +32,32 @@ const addUser = (req, res) => {
     });
 };
 
-module.exports = { addUser };
+const signIn = (req, res) => {
+  const { email } = req.body;
+
+  const token = signToken(email);
+
+  logger.info('Password OK, token sended.');
+  return res.status(200).send({ auth: true, token });
+};
+
+const listUsers = async (req, res) => {
+  const { page } = req.query;
```

Comment:
Destructure page and limit in the same line

`const { page, limit } = req.query;`